examples/terraform-bootstrap/templates/.github/workflows/terraform.yml

```diff
@@ -0,0 +1,46 @@
+{{- if .Variables.CREATE_GITHUB_ACTIONS_PIPELINE -}}
+name: Terraform CI/CD
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "terraform/**"
+      - ".github/workflows/terraform*.yml"
+
+env:
+  TERRAFORM_VERSION: 1.5.7
+
+jobs:
+  {{- range $i, $env := .Variables.ENVIRONMENTS }}
+  {{- if (gt $i 0) }}
+    {{/* Add empty line if not the first job */ -}}
+  {{ end }}
+  build-{{ $env.NAME }}:
+    name: Build for {{ $env.NAME | upper }}
+    uses: ./.github/workflows/terraform-plan.yml
+    with:
+      ENVIRONMENT: {{ $env.NAME }}
+      TERRAFORM_VERSION: {{ "${{ env.TERRAFORM.VERSION }}" }}
```

Comment:
@majori this should be `env.TERRAFORM_VERSION` with an underscore probably, right?